java/core/src/test/java/co/worklytics/psoxy/storage/StorageHandlerTest.java

```diff
@@ -72,7 +74,10 @@ void buildRequest() {
         when(config.getConfigPropertyAsOptional(eq(BulkModeConfigProperty.OUTPUT_BASE_PATH)))
             .thenReturn(Optional.empty());
 
-        StorageEventRequest request = handler.buildRequest(mock(InputStreamReader.class), "bucket-in", "directory/file.csv", handler.buildDefaultTransform());
+
+
+        StorageEventRequest request = handler.buildRequest(mock(InputStreamReader.class),
```

Comment:
I'd try testing with a big file to see how it behaves mem wise, but not a big deal

Reply:
yeah, will have to; hard to replicate in tests exactly, but could try